articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Personal Growth - Why every Project Should Have Gardeners
+---
+# {{page.title}}
+
+Personal Growth - Why every Project Should Have Gardeners
+
+Everyone started small. Even the tallest and strongest tree started as a seed and over time, thanks to a lot of sunlight, rainfall and good soil it became a deep-rooted, reliable and sturdy tree. And every year in the summertime it offers shade and fruits to anyone who appreciates it.
+Very much the same also applies to us, people who are involved in project work, where we try to give our best effort to make the project a success. 
+
+We work in projects as a subject matter expert, as an experienced project manager or as a newcomer that will need support from the team to grow to a new leader. Roles and tasks are assigned based on experiences we made in previous assignments. Experiences from former projects are our soil. We are able to grow on new and demanding tasks but will fail on overwhelming tasks. 
```

Comment:
"to grow to a new leader" -> das verstehe ich nicht. Ist das gemeint: "that will need support from the team to grow to a new level" oder "that will need support from the team to grow into the project" oder "that will need support from the team to grow personally". 

Kürzungspotential / Sprache: "Roles and tasks are assigned based on experiences we made in previous assignments. Experiences we made ..." -> "Roles and tasks are assigned based on our experience. Experience we made ..."